.github/workflows/golangci-lint.yml

```diff
@@ -21,8 +21,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v3
-        with:
-          go-version: '<1.17'
```

Comment:
This needs to stay here but get bumped to `<1.18` until we update the linter version